tests/python_functional/src/syslog_ng_config/statements/sources/source_writer.py

```diff
@@ -25,15 +25,15 @@
 
 
 class SourceWriter(object):
-    def __init__(self, IOClass):
-        self.__IOClass = IOClass
+    def __init__(self, driver_io_cls):
+        self.__driver_io_cls = driver_io_cls
```

Comment:
Optional, probably matter of taste: but I think the original was better. Camelcase hints users that it is a class type and not an instance.